### DISCUSSION TITLE
Map ports

### DISCUSSION THREAD

**Discussion by V-ktor** [redeclipse/base#1132] [Maps and Content Showcase]
I ported my two race maps now. They all have a day and night version.
I updated Darkness as well to have less lights (but the ent count is quite high as there are multiple lights for day or night only).

![20180120104223](https://user-images.githubusercontent.com/12966454/35182348-50d501fa-fdd3-11e7-9195-f6d162f087f3.jpg)
![20180120111518](https://user-images.githubusercontent.com/12966454/35182351-548cc76a-fdd3-11e7-8b33-03ed771d7cab.jpg)
![20180120105118](https://user-images.githubusercontent.com/12966454/35182355-69a35c22-fdd3-11e7-8f66-1c5adc0d0705.jpg)
![20180120111740](https://user-images.githubusercontent.com/12966454/35182361-94340b76-fdd3-11e7-9b05-000ffa5c7e59.jpg)
![20171110122706](https://user-images.githubusercontent.com/12966454/35182399-4e185fec-fdd4-11e7-852e-bfe01251baf9.jpg)

**Map files:**
[absorption.zip](https://github.com/red-eclipse/world/files/1648705/absorption.zip)
[darkness.zip](https://github.com/red-eclipse/world/files/1648706/darkness.zip)
[decomposition.zip](https://github.com/red-eclipse/world/files/1648707/decomposition.zip)

**Comment by ZeroKnight** [comment]
I quite like that last screenshot of Darkness

**Comment by shirepirate** [comment]
you have write access to /maps already, so update your content as needed. looking very nice though

**Comment by no-lex** [comment]
Make sure it follows the quality standards in https://github.com/red-eclipse/maps/issues/48 and update the issue to reflect the changes if you do merge them. Alternatively, you can submit a pull request and @acerspyro or I can review it before merging.

**Comment by V-ktor** [comment]
I have some problems with Nova. The frame rate is really low in the middle of the map. However it goes back to 60 if I disable vsync or delete all lights (deleting half of them does not work) for some reason.
[nova.zip](https://github.com/red-eclipse/world/files/1670411/nova.zip)
![20180127184445](https://user-images.githubusercontent.com/12966454/35475136-615d4994-0399-11e8-98f3-3d50bbe27e7e.jpg)

**Comment by SniperGoth** [comment]
Doesn't seem to affect me.

Vsync 1
![20180127173308](https://user-images.githubusercontent.com/24359960/35475586-53e69678-0388-11e8-8365-b36a732bb756.png)

Vsync 0 
![20180127173319](https://user-images.githubusercontent.com/24359960/35475590-6076280e-0388-11e8-9eb0-0d403ae0ac54.png)

Unless i use the outline, then things get dirty.

Vsync 1, outline 1 
![20180127173612](https://user-images.githubusercontent.com/24359960/35475607-aae743be-0388-11e8-93ae-0c0f55407e0b.png)

Vsync 0, outline 1 

![20180127173728](https://user-images.githubusercontent.com/24359960/35475611-c37d6a98-0388-11e8-858d-78041129d8c4.png)

**Comment by V-ktor** [comment]
It happened without outlines as well.
I just remembered that I enabled msaa. After disabling msaa this does not happen anymore.

**Comment by V-ktor** [comment]
Wet (with lightning!)
I'm not sure how much the lightfx affects performance. The rate of lightning is once every 11.6 seconds.
Curved geometry has more verts and trak4 textures were replaced.
[wet.zip](https://github.com/red-eclipse/world/files/1713253/wet.zip)
![20180210110525](https://user-images.githubusercontent.com/12966454/36061000-2418a1da-0e54-11e8-8009-00c97e905202.jpg)
![20180210110619](https://user-images.githubusercontent.com/12966454/36061009-49884826-0e54-11e8-9a7d-8d92239f9cbd.jpg)
